waku-bindings/src/node/mod.rs

```diff
@@ -29,7 +29,6 @@ pub use config::{GossipSubParams, WakuLogLevel, WakuNodeConfig, WebsocketParams}
 pub use discovery::{waku_discv5_update_bootnodes, waku_dns_discovery, DnsInfo};
 pub use peers::{Protocol, WakuPeerData, WakuPeers};
 pub use relay::{waku_create_content_topic, waku_default_pubsub_topic};
-pub use store::{waku_local_store_query, waku_store_query};
```

Comment:
Is this intentional? 
I saw that waku_local_store_query is used in this file.

Reply:
yeah, but i'm refering to it with the package name : `store::waku_local_store_query`. If i leave the import there i could just use `waku_local_store_query`.